src/xcode/ENA/ENAUITests/Home/ENAUITestsHome.swift

```diff
@@ -36,7 +36,7 @@ class ENAUITests_01_Home: XCTestCase {
 	}
 
 	func test_0010_HomeFlow_medium() throws {
```

Comment:
What is your opinion do we really need to have automated tests with different Fontsizes?

Reply:
Yes. There have been already a lot of them before